g1ant-language/refining-your-code.md

```diff
@@ -15,21 +15,21 @@ excel.getvalue row ♥rownumber colname f result ♥cell6
 
 There is the `♥rownumber` variable which takes its value from the `for` loop, so it stays the same in all 6 rows of the procedure. How can you make the other two arguments change incrementally in the same line as the first one? By employing a list variable and another loop.
 
-Let’s start with the `♥cell` variable. Instead of creating a new variable for each cell, you could just create a list variable and add each cell’s content to this list as a value of a subsequent element. The `♥cell` list variable would look like this: `⟦cell1❚cell2❚cell3❚cell4❚cell5❚cell6⟧`. In order to move through the list elements, you would use the good old `foreach` loop or ordinary `for` loop and incremental indexing in the list variable.
+Let’s start with the `♥cells` variable. Instead of creating a new variable for each cell, you could just create a list variable and add each cell’s content to this list as a value of a subsequent element. The resulting `♥cells` list variable in this case would look like this: `cell1❚cell2❚cell3❚cell4❚cell5❚cell6`. In order to move through the list elements, you would use the good old `foreach` loop or ordinary `for` loop and incremental indexing in the list variable.
 
 The second approach would be even better, since you could apply the same incremental indexing in column addressing. “Wait”, you can ask, “how come? There are letters as values for the `colname` argument!”. Yes, you are absolutely right! This is why you won’t use that argument. Instead, you can refer to a more practical `colindex` argument, which operates on column numbers, not letters.
 
-Before calling `GetValues` and `SetValues` procedures, you have to declare the `♥cell` variable and put it somewhere at the beginning of your script, after both `♥datafile` variables, for example:
+Before calling `GetValues` and `SetValues` procedures, you have to create an empty `♥cells` list variable and put it somewhere at the beginning of your script, after both `♥datafile` variables, for example:
 
 ```G1ANT
 ♥datafile1 = ‴♥environment⟦USERPROFILE⟧\Documents\Data\data.xlsx‴
 ♥datafile2 = ‴♥environment⟦USERPROFILE⟧\Documents\Data\data2.xlsx‴
-♥cell = ⟦cell1❚cell2❚cell3❚cell4❚cell5❚cell6⟧
+list.create ♥cells
```

Comment:
this is incorrect, have you tried running such code? :p 
should be: `list.create result ♥cells`

Reply:
```suggestion
list.create result ♥cells
```